src/utils/storage/index.ts

```diff
@@ -83,13 +85,15 @@ const StorageUtils = {
   otherProfileData: storageBuilder(StorageKeysEnum.OtherProfileData),
   lastPromptNotification: storageBuilder(StorageKeysEnum.LastPromptNotification),
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
+  blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
+  blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `blockingStatus` and `blockedStatus` to the `StorageUtils` object is a good way to manage these new keys. However, it's important to ensure that these keys are used consistently throughout the application to avoid any potential issues with data inconsistency or retrieval errors.

```diff
+  blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
+  blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->